recipes/mms-python-adapter/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "mms-python-adapter" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dded9d59246be03a8fb1f299b1b70b2dbf85dcea87076316e65d23f7bbc7fa7d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - mms-python-client <4
+
+test:
+  imports:
+    - mms_python_adapter
+
+about:
+  home: http://openmbee.org
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: QOL wrapper for mms-python-client
+
+extra:
+  recipe-maintainers: 
+    - HuiJun
```

Comment:
@HuiJun are you OK with being a maintainer?

Reply:
LGTM